pandas/core/internals/blocks.py

```diff
@@ -266,6 +268,7 @@ def getitem_block(self, slicer: slice | npt.NDArray[np.intp]) -> Block:
 
         new_values = self._slice(slicer)
         refs = self.refs if isinstance(slicer, slice) else None
+        refs = self.refs if isinstance(slicer, slice) else None
```

Comment:
small leftover from the merge

Reply:
Yep, thx, I am already going over them. There seem to be a couple of things off